### PR TITLE
Dropdown + FloatingDropdown divider

### PR DESCRIPTION
**Pull request by jonathanadler**
Added support for a divider in the Dropdown and FloatingDropdown components

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_429/

File: src/components/Dropdown/Dropdown.st.css [hunk 1]
```diff
@@ -137,3 +137,9 @@
 .root:alignment(center) DropdownOption {
     justify-content: center;
 }
+
+.divider {
+    margin: 10px 13px !important;
+    width: auto !important;
+    opacity: 0.2 !important;
+}
```
Comment (sivanharel) [inline src/components/Dropdown/Dropdown.st.css:145]: Why do you need important?
Are you overriding the wix-ui-core styling?

**Review by sivanharel — COMMENTED** [review]
Looks good.
when running it loccaly I am getting this error:
src/components/Dropdown/DropdownNativeSelect.tsx(52,20): error TS2339: Property 'divider' does not exist on type 'DropdownOptionProps[]'.
And I wrote a small question.

**Review by sivanharel — APPROVED** [review]
I see that you fixed the issue :)

File: src/components/Dropdown/Dropdown.st.css (hunk 1 above)
Comment (jonathanadler) [reply src/components/Dropdown/Dropdown.st.css:145]: That's right :( 
I'll see if there's a way to remove them

Comment (jonathanadler) [reply src/components/Dropdown/Dropdown.st.css:154]: fixed

**Pull request merged**